modules/codegen/src/main/scala/com/twilio/guardrail/terms/SwaggerTerm.scala

```diff
@@ -170,7 +175,26 @@ case class RouteMeta(path: String, method: HttpMethod, operation: Operation, sec
             p.setExtensions(Option(schema.getExtensions).getOrElse(new java.util.HashMap[String, Object]()))
             p
         }
+      })
+      .traverse[State[ParameterCountState, ?], Parameter] { p =>
```

Comment:
Out of curiosity, why the `?` here?

Reply:
`?` comes from https://github.com/typelevel/kind-projector
It's a simple macro that expands things of the shape `F[?]` into `({ type λ[A] = F[A] })#λ`, letting us write `.traverse[State[Foo, ?], Bar]` instead of `.traverse[({ type λ[A] = State[Foo, A] })#λ, Bar]`. Very handy.

Reply:
It's very common in FP libraries, fwiw.

Reply:
Ah, right.  I remembered the kind-projector thing, but not what it expanded to.